EquiCLIP/main_lambda_equitune.py

```diff
@@ -105,13 +105,12 @@ def main(args):
                 print(f"Learning lambda weights: {i}/{args.num_prefinetunes}")
             # zeroshot prediction
             # add weight_net save code for the best model
-
-            # evaluating for only 50 steps using val=True
+            val = False if args.full_val_pf else True # evaluating for only 50 steps using val=True if full_val_pf is False
```

Comment:
maybe we can omit that and stick with val=True?

Reply:
i dont mind